C-Plus-Plus/cp/ratInAMaze.cpp

```diff
@@ -0,0 +1,124 @@
+/*There is a rat who is at the starting point of a matrix and wants to reach
+ to the ending point i.e at the last coordinate of the matrix.
+ 1 denotes the point where it is possible for the mouse to go and
+ 0 denotes that the mouse cannot reach at that point. */
```

Comment:
Can you add more details about the Problem here 😅 

Also if possible add a Problem Link on some Online Coding Platform